src/WebApps/SnWebApplication.Api.InMem.TokenAuth/Dockerfile

```diff
@@ -0,0 +1,34 @@
+#See https://aka.ms/containerfastmode to understand how Visual Studio uses this Dockerfile to build your images for faster debugging.
+
+FROM mcr.microsoft.com/dotnet/aspnet:3.1 AS base
+WORKDIR /app
+EXPOSE 80
+EXPOSE 443
+
+FROM mcr.microsoft.com/dotnet/sdk:3.1 AS build
+WORKDIR /src
+COPY ["WebApps/SnWebApplication.Api.InMem.TokenAuth/SnWebApplication.Api.InMem.TokenAuth.csproj", "WebApps/SnWebApplication.Api.InMem.TokenAuth/"]
+COPY ["OData/SenseNet.OData.csproj", "OData/"]
```

Comment:
Do we need to copy individual csproj files?

Reply:
don't have to, vs creates in this form by default, I'll change this